backend/src/thirdParties/github/githubService.js

```diff
@@ -0,0 +1,200 @@
+const service = {
+  name: 'github',
+  triggers: [
+    {
+      name: 'star',
+      data: {
+        action: 'created',
+        starred_at: '2019-05-15T15:20:40Z',
+        repository: {
+          id: 186853002,
+          node_id: 'MDEwOlJlcG9zaXRvcnkxODY4NTMwMDI=',
+          name: 'Hello-World',
+          full_name: 'Codertocat/Hello-World',
+          private: false,
+          owner: {
+            login: 'Codertocat',
+            id: 21031067,
+            node_id: 'MDQ6VXNlcjIxMDMxMDY3',
+            avatar_url: 'https://avatars1.githubusercontent.com/u/21031067?v=4',
+            gravatar_id: '',
+            url: 'https://api.github.com/users/Codertocat',
+            html_url: 'https://github.com/Codertocat',
+            followers_url: 'https://api.github.com/users/Codertocat/followers',
+            following_url:
+              'https://api.github.com/users/Codertocat/following{/other_user}',
```

Comment:
`'https://api.github.com/users/Codertocat/following{/other_user}'`
Can you please explain this? especially curly braces part

Reply:
This is a sample format of the event we will receive from Github when listening to star events. We need this to show the suggestion to user when creating Zapp.

Reply:
Also, this API is used to determine whether a `Codertocat` follows a any other user.